gradle/libs.versions.toml

```diff
@@ -44,6 +44,14 @@ micronaut-spring = "5.10.0"
 micronaut-docs = "2.0.0"
 sonatype-scan = "3.0.0"
 
+# javascript libraries
+js-swagger-ui="5.20.5"
+js-openapi-explorer="2.2.733"
+js-redoc="2.4.0"
+js-rapidoc="9.3.8"
+js-rapipdf="2.2.1"
+js-scalar="1.28.11"
```

Comment:
is there a way to get dependency scanners to pick this up?

Reply:
I think one option would be to create take nodejs lock files or somethinig. Not very familiar with this.